scripts/check-env.sh

```diff
@@ -43,4 +47,41 @@ fi
 [ "$(go env CGO_ENABLED)" = '1' ] \
 	|| fail "CGO_ENABLED should be set to '1'"
 
+# create a temp go file
+LIBCHECK=$(mktemp)
+mv "${LIBCHECK}" "${LIBCHECK}".go
+LIBCHECK=${LIBCHECK}.go
+
+# check for packages using a compile test
+cat << EOF > "${LIBCHECK}"
+package main
+
+/*
+#include <rados/librados.h>
+#include <rbd/lirbd.h>
+*/
+import "C"
+
+func main() {
+	_ = C.LIBRADOS_VERSION_CODE
```

Comment:
You can check `/usr/include/rbd/librbd.h` and use one of the `#define`'s from it too. Building most likely fails if the file is missing, so it is not strictly required.